README.md

```diff
@@ -2,9 +2,21 @@
 
 > A tool for rewriting semgrep matches using externals tools
 
+### Installation:
+
+If you have a Go toolchain installed, you can build the `semgrepx` binary like this:
+
+```sh
+git clone https://github.com/icholy/semgrepx
+cd semgrepx
+go build
+```
+You can then run the binary as `./semgrepx`
```

Comment:
Go binaries are usually installed using `go install` like this:

```
go install github.com/icholy/semgrepx@latest
```

They're placed in your `GOBIN` directory, which defaults to `~/go/bin`.

Reply:
TIL!

Reply:
https://til.simonwillison.net/go/installing-tools